sockeye/inference.py

```diff
@@ -54,6 +60,7 @@ class InferenceModel(model.SockeyeModel):
     :param decoder_return_logit_inputs: Decoder returns inputs to logit computation instead of softmax over target
                                         vocabulary.  Used when logits/softmax are handled separately.
     :param cache_output_layer_w_b: Cache weights and biases for logit computation.
+    :param json_input: Input is in JSON format
```

Comment:
Missing period at end (I'm learning from @fhieber  ;-) )

Reply:
Actually the docstring is bad since there is no corresponding parameter :)